announcements/1.13.0.md

```diff
@@ -0,0 +1,93 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.13.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### OpenGL ES 3.2 support
+
+After supporting OpenGL ES 3.1 only for a long time Anbox Cloud finally adds full OpenGL ES 3.2 support
+for NVIDIA GPUs and software rendering. This provides a increased compatibility with available Android
+games.
+
+### Direct rendering for AMD and Intel GPUs
+
+Since the beginning of Anbox Cloud graphics APIs were exposed to the Android container through an
+translation layer and actual execution of commands happened on the outside. With 1.13, Anbox Cloud
+now enables direct GPU access inside the Android container for AMD and Intel GPUs based on the
+open source [Mesa driver](https://mesa3d.org/). This provides better performance and stability and
+increased compatibility.
+
+Currently OpenGL ES up to version 3.2 is supported. Vulkan support will be added in Anbox Cloud 1.14.
+
+In Anbox Cloud 1.14 software rendering will be migrated to direct rendering too.
```

Comment:
```suggestion
In Anbox Cloud 1.14, software rendering will be migrated to direct rendering too.
```

Reply:
It's implicit in the statement and the heading, but since the first section talks both about GPUs and software rendering, I  missed that this feature isn't for software rendering yet and was surprised to read this here.
Maybe we can say it more directly:
```suggestion
Currently, direct rendering is available only when using GPUs. In Anbox Cloud 1.14, software rendering will be migrated to direct rendering too.
```